tests/README.md

```diff
@@ -2,9 +2,25 @@
 - baseclasses
 - testflo
 
-# Instructions
-- To train regression tests: `testflo reg_tests -m train`
-- To run regression tests: `testflo reg_tests`
-- To run unit tests: `testflo unit_tests`
-- To run regression and unit tests: `testflo reg_tests unit_tests`
-- To run individual test: `testflo reg_tests/test1.py`
\ No newline at end of file
+
+# Instructions and examples
+- To train all regression tests with specific function signature: `testflo reg_tests -m "train_*"`
+- To train all regression tests in specific testfile either works: `cd reg_tests; testflo test_DVGeometry.py -m "train_*"` or `testflo reg_tests/test_DVGeometry.py -m "train_*"` 
+- To train specific regression test in specific testfile either works: `cd reg_tests; testflo test_DVGeometry.py -m "test_1` or `testflo reg_tests/test_DVGeometry.py -m "train_1"`
+
+- To test all regression tests: `testflo reg_tests`
+- To test all regression tests with specific function signature: `testflo reg_tests -m "test_*"`
+- To test all regression tests in specific testfile either works: `cd reg_tests; testflo test_DVGeometry.py -m "test_*"` or `testflo reg_tests/test_DVGeometry.py -m "test_*"` 
+- To test specific regression test in a specific testfile either works: `cd reg_tests; testflo test_DVGeometry.py -m "test_1` or `testflo reg_tests/test_DVGeometry.py -m "test_1"`
+
+- To enable screen output: use `-s` argument
+- To change number of cores to 1 for running tests: use `-n 1`
+
+## Other ways to run
+The tests are also compatiable with general ```unittesting``` python framework and can be call individually using, 
+
+```python unit_tests/test_basics.py```
```

Comment:
Will this work with MPI, or are you simply running the same test but with one proc?

Reply:
It wouldn't work with MPI, it uses just one proc

Reply:
Right, I don't think it would make sense to have that there then, since it's not going to run the test with the correct configuration. Since we can call individual tests with testflo we don't need to run it with the unittest framework directly.

Reply:
I found using unittesting directly to be very valuable. Specifically when I wanted to call one sub test for debugging (test_tutorial_wing.TestFunctionals.test_jac_vec_prod_bwd for example).
Testflo might be able to do this but I haven't figured it out. 

would you support rewording to something like ...

> The tests can also be called without multi-core execution using unittesting directly with
```python unit_tests/test_basics.py```
or any other way to call the unittesting package. 
This method of running the tests should only be used for debugging the tests since it is not compatible with MPI and will not catch issues with parallel execution.

Reply:
It is possible to run individual tests. For example in pyoptsparse, you could run something like `testflo test/test_hs015.py:TestHS15.test_snopt` to run that specific test. We should definitely explain this syntax in our README though.

Would that be sufficient to replace the unittest stuff?

Reply:
Yeah that would work. I'll remove any reference to unittesing